test/snippet/utility/tuple/pod_tuple.cpp

```diff
@@ -1,6 +1,9 @@
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/utility/tuple/pod_tuple.hpp>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmissing-braces"
```

Comment:
Are you testing GCC & libc++ or clang with libc++?
Does this also happen with clang?

Reply:
Usually clang only. Seems to be the one bogus thing they have... 
https://github.com/seqan/seqan3/blob/fc8f4f7db4a901380d3b7cdf8acad20aecc85747/test/unit/utility/tuple/pod_tuple_test.cpp#L14-L15

Reply:
Managed to fix the warning, rewrote the commit